### PR TITLE
Handle create cat with db

### PR DESCRIPTION
**Pull request by benbeckman2001**

### CODE REVIEW

File: src/entities/user.entity.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity()
+export class User {
+  @PrimaryGeneratedColumn()
+  id: string;
+  @Column()
```
Comment (benbeckman2001) [inline src/entities/user.entity.ts:7]: look quite ok

Comment (OptimusPrime2001) [reply src/entities/user.entity.ts:7]: yeb

**Review by OptimusPrime2001 — COMMENTED** [review]
look so good

File: src/app.module.ts [hunk 2]
```diff
@@ -1,38 +1,39 @@
-import { Module } from '@nestjs/common';
```
Comment (OptimusPrime2001) [inline src/app.module.ts:1]: ffff

**Review by OptimusPrime2001 — APPROVED** [review]
sdaf

**Pull request merged**